src/sdkV2.ts

```diff
@@ -408,6 +425,75 @@ export default class CegaEvmSDKV2 {
    * USER FACING METHODS
    */
 
+  // ====== lpCega Offramp (Vault Token Market) code starts here
+
+  async getOfframpFeeBps(): Promise<number> {
+    const cegaEntry = await this.loadCegaEntry();
+    return cegaEntry.getOfframpFeeBps();
+  }
+
+  async getOrderData(maker: EvmAddress, orderHash: string): Promise<GetOrderDataResponse> {
+    const cegaEntry = await this.loadCegaEntry();
+    return cegaEntry.getOrderData(maker, orderHash);
+  }
+
+  async hashOrder(order: LpCegaOfframpOrder): Promise<string> {
+    const cegaEntry = await this.loadCegaEntry();
+    return cegaEntry.hashOrder(order);
+  }
+
+  async signOrder(
+    cegaEntry: ethers.Contract,
+    order: LpCegaOfframpOrder,
+    chainId: number,
+  ): Promise<string> {
+    const signer = this._signer;
+
+    if (!signer) {
+      throw new Error('Signer not defined');
+    }
+
+    const domain = {
+      name: 'Cega Offramp Entry',
+      version: '1',
+      chainId,
+      verifyingContract: cegaEntry.address,
+    };
+
+    return (signer as ethers.providers.JsonRpcSigner)._signTypedData(
+      domain,
+      TYPES_OFFRAMP_ORDER,
+      order,
+    );
+  }
+
+  async fillOrder({ order, swapMakingAmount }: FillOrderParams): Promise<FillOrderResponse> {
```

Comment:
shouldn't this just be a `TransactionResponse`?

Reply:
I'm not sure, though, but I can see the outputs mentioned in the response. Either @poegva can confirm, or we can confirm once we are able to interact with the contract.

```
"outputs": [
        {
          "internalType": "uint256",
          "name": "swapTakingAmount",
          "type": "uint256"
        },
        {
          "internalType": "bytes32",
          "name": "orderHash",
          "type": "bytes32"
        }
      ]
```

Reply:
see `fcnCreateVault` and `dcsCreateVault`. those also have outputs in the ABI, but I don't think we actually have access to the outputs for transactions. the "outputs" I think are only accessible to other caller functions in the smart contract

the request usually just responds with the transaction information. it doesn't wait to determine if the transaction was successfully mined or not

Reply:
agreed, updated the code